calico/getting-started/kubernetes/self-managed-public-cloud/gce.mdx

```diff
@@ -35,21 +35,21 @@ You will need at least one VM to serve as a control plane node and one or more w
 
 The following worked example creates a single control node and three workers on a dedicated virtual private network (VPC). Adjust the example as needed for your requirements. Consider a dedicated infrastructure management tool like [Terraform](https://www.terraform.io/) for managing cloud resources. (This example is adapted from [Kubernetes the Hard Way](https://github.com/kelseyhightower/kubernetes-the-hard-way/blob/master/docs/03-compute-resources.md).)
 
-**Create the VPC**
+1. Create the VPC.
```

Comment:
If you're adding step numbers, you'll need to indent the code blocks and other elements so they align with the steps.

Reply:
Done.